deploy/config/archiver.conf

```diff
@@ -0,0 +1,58 @@
+##### Catalog Service #####
```

Comment:
is this archiver.conf needed by the dispatcher?

Reply:
same for the others conf files

Reply:
done